### PR TITLE
Fixed console errors that appear while running tests

### PR DESCRIPTION
**Pull request by jameskim08**
Removed deprecated use of sinon.spy() with three parameters
Stubbed init to avoid fetch calls on the client

### CODE REVIEW

File: test/unit/cart.js [hunk 1]
```diff
@@ -300,23 +299,29 @@ describe('Cart class', () => {
   });
 
   describe('init', () => {
+    let superInitStub;
+    let fetchMoneyFormatStub;
+    beforeEach(() => {
+      superInitStub = sinon.stub(Component.prototype, 'init').resolves({model: {lineItems: [{id: 123, quantity: 5}]}});
+      fetchMoneyFormatStub = sinon.stub(cart, 'fetchMoneyFormat').returns(Promise.resolve('test'));
```
Comment (melissaluu) [inline test/unit/cart.js]: Let's be consistent and use the `resolves()` function instead of the `returns(Promise....)`. 

Also, if we're not testing the resolved promise's return, let's leave it as an empty promise return.
